policy_validator.c

```diff
@@ -7,11 +7,11 @@ static PolicyValidatorConfig policy_validator;
 
 CPURISCVState* policy_validator_hack_env;
 
-static inline target_ulong policy_validator_reg_reader(uint32_t reg_num)
+static inline uint64_t policy_validator_reg_reader(uint32_t reg_num)
```

Comment:
Won't `target_ulong` change depending on architecture to what we need?

Reply:
It does, which is one of the reasons we originally needed two interface definitions.  Fixing the size helps commonize the interface.